enos/enos-samples-ce-build.hcl

```diff
@@ -258,7 +296,13 @@ sample "build_ce_linux_arm64_zip" {
       arch            = ["arm64"]
       artifact_source = ["crt"]
       artifact_type   = ["bundle"]
+      consul_edition  = ["ce"]
       edition         = ["ce"]
+
+      # arm64 AMIs not offered for all versions of Leap
```

Comment:
For posterity: I did not, in fact, understand the behavior correctly! The sampling feature creates an "additional" filter that is applied to the entire universe of possible scenarios as defined in the scenarios themselves, so any exclusions would already be excluded. Therefore, you would not have a situation in which a sampling filter would cause a "matching filter not found" error to be thrown.